src/main/java/com/artipie/settings/MetricsContext.java

```diff
@@ -53,7 +59,7 @@ public boolean enabled() {
      */
     public Optional<Pair<String, Integer>> endpointAndPort() {
         Optional<Pair<String, Integer>> res = Optional.empty();
-        final YamlMapping metrics = this.meta.yamlMapping("metrics");
+        final YamlMapping metrics = this.meta.yamlMapping(MetricsContext.METRICS);
```

Comment:
We should parse `yaml` only one time. 
Currently, we have to parse the same `yaml` many times when every method is called.